int32 id
int32 xmin
int32 ymin
int32 xmax
int32 ymax
float32 confidence
int32 classNumber
string className
float32 depthAtCenter